cmake/modules/FindTBB.cmake

```diff
@@ -164,15 +164,15 @@ if(NOT TBB_FOUND)
     endif()
   elseif(CMAKE_SYSTEM_NAME STREQUAL "Linux")
     # Linux
-    set(TBB_DEFAULT_SEARCH_DIR "/opt/intel/tbb")
+    set(TBB_DEFAULT_SEARCH_DIR "/opt/intel/oneapi/tbb/latest")
```

Comment:
We cannot move USD unconditionally to oneapi - it must be buildable with tbb 2020 or oneapi fr a suitable adoption period.